packages/react-components/react-color-picker-preview/library/src/components/ColorSlider/ColorSlider.test.tsx

```diff
@@ -34,4 +34,105 @@ describe('ColorSlider', () => {
       </div>
     `);
   });
+
+  it('renders horizontal ColorSlider correctly', () => {
+    const { container } = render(<ColorSlider defaultValue={50} min={0} max={100} />);
+    expect(container).toMatchInlineSnapshot(`
```

Comment:
wondering how can one determine that this test covers that something renders horizonally/vertically...

after further investigation it seems only "differentiator" is css variable `--fui-Slider--direction: 180deg`, if that's correct can we rather not use snapshots and test only what makes sense ?

Reply:
Yeah, I'll remove it. We'll test vertical/horizontal in VR tests

Reply:
It's ok to test it here if needed, what I was saying is to not use snapshots if not necessary rather scoped approach like these other spec you wrote.